### PR TITLE
XD60 Finnish/Swedish layout

### PR DESCRIPTION
**Pull request by pallarim**
## Description

For Xiudi's XD60, added layout for Finnish/Swedish language. Tested with XD60 rev3.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xiudi/xd60/keymaps/finnish/keymap.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/xiudi/xd60/keymaps/finnish/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/xiudi/xd60/keymaps/finnish/keymap.c (hunk 1 above)
Comment (pallarim) [reply keyboards/xiudi/xd60/keymaps/finnish/keymap.c:4]: @drashna, Thank you for the feedback. License header has now been added.

**Review by drashna — APPROVED** [review]

File: keyboards/xiudi/xd60/keymaps/finnish/readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+![Base layer](https://i.imgur.com/TrChS0N.png)
+
+```json
+["Esc","!\n1","\"\n2","#\n3","¤\n4","%\n5","&\n6","/\n7","(\n8",")\n9","=\n0","?\n+","`\n´","Fn",{w:1},"Back space"],
+[{w:1.5},"Tab","Q","W","E","R","T","Y","U","I","O","P","Å","^\n¨",{x:0.25,w:1.25,h:2,w2:1.5,h2:1,x2:-0.25},"Enter"],
+[{w:1.75},"Caps Lock","A","S","D","F","G","H","J","K","L","Ö","Ä","*\n'"],
+[{w:1.25},"Shift","<\n>","Z","X","C","V","B","N","M",";\n,",":\n.",{w:1.75},"Shift","Up","_\n-"],
+[{w:1.5},"Ctrl",{w:1},"Win",{w:1.25},"Alt",{a:7,w:6.25},"",{a:4,w:1},"AltGr",{w:1},"Ctrl",{w:1},"Left",{w:1},"Down","Right"]
+```
+
```
Comment (fauxpark) [inline keyboards/xiudi/xd60/keymaps/finnish/readme.md:18]:
```suggestion
```

File: keyboards/xiudi/xd60/keymaps/finnish/keymap.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+#include "keymap_finnish.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  // 0: Base Layer
+  LAYOUT_all(
+      KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    FI_PLUS,  FI_ACUT,  MO(1), KC_BSPC,    \
+      KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    FI_ARNG,  FI_DIAE,           KC_NO,   \
+      KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    FI_ODIA, FI_ADIA, FI_QUOT,             KC_ENT,    \
+      KC_LSFT, FI_LABK,   KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    FI_MINS,      \
```
Comment (fauxpark) [inline keyboards/xiudi/xd60/keymaps/finnish/keymap.c:14]:
```suggestion
      KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    FI_PLUS,  FI_ACUT,  MO(1), KC_BSPC,
      KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    FI_ARNG,  FI_DIAE,           KC_NO,
      KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    FI_ODIA, FI_ADIA, FI_QUOT,             KC_ENT,
      KC_LSFT, FI_LABK,   KC_Z,    KC_X,    KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    FI_MINS,
```

File: keyboards/xiudi/xd60/keymaps/finnish/keymap.c [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      KC_LCTL, KC_LGUI, KC_LALT,                          KC_SPC,                          KC_RALT, KC_RCTL,    KC_LEFT, KC_DOWN,  KC_RIGHT),
+
+  // 1: Function Layer
+  LAYOUT_all(
+      RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_F13,   KC_F14,    \
+      KC_NO,   KC_WH_U, KC_UP,   KC_WH_D, KC_BSPC,KC_HOME,KC_CALC,KC_NO,  KC_INS, KC_NO,   KC_PSCR, KC_SLCK,  KC_PAUS,           KC_DEL,    \
+      KC_NO,   KC_LEFT, KC_DOWN, KC_RIGHT,KC_DEL, KC_END, KC_PGDN,KC_NO,  KC_NO,  KC_NO,   KC_HOME, KC_PGUP,  KC_NO,             KC_ENT,    \
+      KC_LSFT, KC_NO,   KC_NO,   KC_APP,  BL_STEP,KC_NO,  KC_NO,  KC_VOLD,KC_VOLU,KC_MUTE, KC_END,  KC_PGDN,  KC_RSFT, KC_PGUP,  KC_INS,      \
```
Comment (fauxpark) [inline keyboards/xiudi/xd60/keymaps/finnish/keymap.c:22]:
```suggestion
      RESET,   KC_F1,   KC_F2,   KC_F3,   KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_F13,   KC_F14,
      KC_NO,   KC_WH_U, KC_UP,   KC_WH_D, KC_BSPC,KC_HOME,KC_CALC,KC_NO,  KC_INS, KC_NO,   KC_PSCR, KC_SLCK,  KC_PAUS,           KC_DEL,
      KC_NO,   KC_LEFT, KC_DOWN, KC_RIGHT,KC_DEL, KC_END, KC_PGDN,KC_NO,  KC_NO,  KC_NO,   KC_HOME, KC_PGUP,  KC_NO,             KC_ENT,
      KC_LSFT, KC_NO,   KC_NO,   KC_APP,  BL_STEP,KC_NO,  KC_NO,  KC_VOLD,KC_VOLU,KC_MUTE, KC_END,  KC_PGDN,  KC_RSFT, KC_PGUP,  KC_INS,
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**